src/humancellatlas/data/metadata/api.py

```diff
@@ -204,7 +204,10 @@ def __init__(self, json: JSON):
         super().__init__(json)
         content = json.get('content', json)
         self.genus_species = {gs['text'] for gs in content['genus_species']}
-        self.disease = {d['text'] for d in content.get('disease', []) if d}
+        try:
+            self.disease = {d['text'] for d in lookup(content, 'disease', 'diseases') if d}
+        except KeyError:
```

Comment:
The `KeyError` could be raised by `d['text']` in which case I don't want to swallow it.

Reply:
Instead of catching the KeyError, please use the `default` keyword argument to `lookup()` that I added today.